.rubocop.yml

```diff
@@ -1,2 +1,5 @@
 require: umts-custom-cops
 inherit_from: .rubocop-no-umts-cops.yml
+
+HasAndBelongsToMany:
```

Comment:
Personally, I like to add comments explaining why we're disagreeing with Rubocop.